### PR TITLE
fixed previous errors and added Angular framework

### PR DESCRIPTION
**Pull request by manproai**
Fixed previous comments and developing database structure and sooner I will change and add all the model schemas

### CODE REVIEW

File: controllers/accountControllers/userAccountController.js [hunk 1]
```diff
@@ -1,117 +1,110 @@
...
-    const userID = req.user.id;
-    const account = await Account.create({
-        user: userID,
-        accountName, 
-        accountCurrency, 
-        accountAmount, 
-        accountDescription, 
-        accountStat,
-        accountCreated: new Date().toGMTString()
+const { Account } = require('../../models/userAccountModel');
```
Comment (AudriusBaltusis) [inline controllers/accountControllers/userAccountController.js:1]: I recommend frontend and backend fixes deliver in separate PRs

File: controllers/accountControllers/userAccountController.js [hunk 2]
```diff
@@ -1,117 +1,110 @@
...
+  }
+  const {
+    accountName,
+    accountCurrency,
+    accountAmount,
+    accountDescription,
+    accountStat,
+  } = req.body;
+
+  const isAccount = await Account.findOne({ accountName });
```
Comment (AudriusBaltusis) [inline controllers/accountControllers/userAccountController.js:18]: `isAccount` name suits for boolean, please looks for better option

File: controllers/accountControllers/userAccountController.js [hunk 3]
```diff
@@ -1,117 +1,110 @@
...
+
+  if (isAccount) {
```
Comment (AudriusBaltusis) [inline controllers/accountControllers/userAccountController.js:20]: Such validation can be done in mongoose schema

**Review by AudriusBaltusis — APPROVED** [review]

File: controllers/accountControllers/userAccountController.js (hunk 1 above)
Comment (manproai) [reply controllers/accountControllers/userAccountController.js:1]: Sorry for the inconvenience. I'll make separate PRs next time

File: controllers/accountControllers/userAccountController.js (hunk 2 above)
Comment (manproai) [reply controllers/accountControllers/userAccountController.js:18]: Understood. Thank you. Fixed that

File: controllers/accountControllers/userAccountController.js (hunk 3 above)
Comment (manproai) [reply controllers/accountControllers/userAccountController.js:20]: I have questions regarding validation. 
Should I make validation only in mongoose, or is it better to do it with both Joi package and mongoose on BE? 
What about front-end validation? If I have front-end validation should I use three of them for safety or isn't it necessary?

Comment (AudriusBaltusis) [reply controllers/accountControllers/userAccountController.js:20]: It depends on case, for example `required` can be validated only on mongoose. Maintaining several different validations is challenging. Frontent validation is required

**Pull request merged**